sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EditEventSteps.java

```diff
@@ -1787,6 +1792,53 @@ public EditEventSteps(
           fillStartDataDE(eventStartDate);
         });
 
+    Then(
+        "^I Verify The Eye Icon opening the Map is disabled in the Edit Event Page",
+        () -> {
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(EYE_ICON);
+          webDriverHelpers.assertElementIsDisabled(EYE_ICON);
+        });
+
+    Then(
+        "^I Verify The Eye Icon opening the Map is enabled in the Edit Event Page",
+        () -> {
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(EYE_ICON);
+          webDriverHelpers.assertElementIsEnabled(EYE_ICON);
+        });
+
+    And(
+        "^I Add the GPS Latitude Values in the Edit Event Page",
+        () -> {
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(GPS_LATITUDE_INPUT_EDIT_EVENT);
+          webDriverHelpers.fillInWebElement(GPS_LATITUDE_INPUT_EDIT_EVENT, "54.34555677");
+          webDriverHelpers.submitInWebElement(GPS_LATITUDE_INPUT_EDIT_EVENT);
+        });
+
+    And(
+        "^I Add the GPS Longitude Values in the Edit Event Page",
+        () -> {
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(GPS_LONGITUDE_INPUT_EDIT_EVENT);
+          webDriverHelpers.fillInWebElement(GPS_LONGITUDE_INPUT_EDIT_EVENT, "23.34555677");
+          webDriverHelpers.submitInWebElement(GPS_LONGITUDE_INPUT_EDIT_EVENT);
+        });
```

Comment:
make a single method and pass both parameters as variables, we shouldn't hardcode them